scripts/create_perf_json.py

```diff
@@ -806,6 +806,12 @@ def is_topdown_row(key: str) -> bool:
                     mgroups.append(group)
                     if group not in self.metricgroups:
                         self.metricgroups[group] = f'Metrics for top-down breakdown at level {level}'
+                tma_perf_metric_l1_performance_cores = ['ICL', 'ICX', 'RKL', 'TGL', 'ADL/RPL', 'SPR']
```

Comment:
To automate this, should this be derived from events?

Reply:
For L1, I think we may check the event TOPDOWN.SLOTS:perf_metrics. But I have no idea how to do it for L2.

Reply:
We cannot use the 'topdown\\-' to distinguish the metrics either, since some TopdownL1 and TopdownL2 are derived from other metrics, e.g., max(1 - (tma_frontend_bound + tma_backend_bound + tma_retiring), 0)", which doesn't include the 'topdown\\-'  event.
Can we merge the hardcode version for now? We can change it if we find a better way later.

Reply:
Thanks. I will merge the pull.

Yes, we can detect the topdown.slots event. That could work for TopdownL1. The problem is how to detect the TopdownL2. The ICX and SPR have the exact same formula for tma_core_bound, max(0, tma_backend_bound - tma_memory_bound). But we only want to TopdownL2 as default on SPR.